capstone.iml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/course2/week1/hello" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/course2/week1/perimeter" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/course2/week1/perimeter_quiz" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="jar://$USER_HOME$/project jar files/courserajava.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES>
+          <root url="jar://$USER_HOME$/project jar files/courserajava.jar!/" />
+        </SOURCES>
+      </library>
+    </orderEntry>
+  </component>
+</module>
```

Comment:
this red-dot indicates that there is no new-line. ensure you have new-line at the end of each file.